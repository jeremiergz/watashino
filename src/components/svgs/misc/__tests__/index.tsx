import DownloadIcon from '@/components/svgs/misc/Download';
import PoIIcon from '@/components/svgs/misc/PoI';
import React from 'react';
import ReactTestRenderer from 'react-test-renderer';

describe('@/components/svgs/misc Test Suite', () => {
  describe('Download Test Suite', () => {
    it('matches snapshot', () => {
      const jsx = <DownloadIcon />;
      expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
    });
  });

  describe('PoI Test Suite', () => {
    it('matches snapshot', () => {
      const jsx = <PoIIcon />;
      expect(ReactTestRenderer.create(jsx).toJSON()).toMatchSnapshot();
    });
  });
});
