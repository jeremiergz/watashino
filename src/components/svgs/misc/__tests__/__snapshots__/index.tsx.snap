// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@/components/svgs/misc Test Suite Download Test Suite matches snapshot 1`] = `
<svg
  className=""
  height={32}
  viewBox="0 0 20 20"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M3 17a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zm3.293-7.707a1 1 0 011.414 0L9 10.586V3a1 1 0 112 0v7.586l1.293-1.293a1 1 0 111.414 1.414l-3 3a1 1 0 01-1.414 0l-3-3a1 1 0 010-1.414z"
  />
</svg>
`;

exports[`@/components/svgs/misc Test Suite PoI Test Suite matches snapshot 1`] = `
<svg
  className=""
  height={32}
  viewBox="0 0 20 20"
  width={32}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M5.05 4.05a7 7 0 119.9 9.9L10 18.9l-4.95-4.95a7 7 0 010-9.9zM10 11a2 2 0 100-4 2 2 0 000 4z"
  />
</svg>
`;
